src/dao/hooks/types.ts

```diff
@@ -9,6 +9,7 @@ export enum DAOQueries {
   FetchCanUnlockNFTToken = "fetchCanUnlockNFTToken",
   FetchBlockedTokenBalance = "fetchBlockedTokenBalance",
   FetchDAOMembers = "fetchDAOMembers",
+  ContractId = "contractId",
```

Comment:
This can just be `Contract = "contract"`. Query keys are recommended to be named by resource: https://tanstack.com/query/v4/docs/react/guides/query-keys.